models/models.go

```diff
@@ -94,7 +97,21 @@ func Setup(c *config.Config) error {
 		return err
 	}
 	// Open our database connection
-	db, err = gorm.Open(conf.DBName, conf.DBPath)
+	i := 0
+	for {
+		log.Warn("waiting for database to be up...")
```

Comment:
One more super small nitpick - can we move this to line 111 (above the `i +=1`)? The benefit is that if the database connection succeeds the first time, then this message won't show up in the logs.

Reply:
done!